src/database-controller/sdk/.dockerignore

```diff
@@ -0,0 +1,63 @@
+.git
+
+# Directory for submitted jobs' json file and scripts
+frameworklauncher/
+
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Runtime data
+pids
+*.pid
```

Comment:
Why so many this file in your pr?

Reply:
I have merged two dockerignore files into one.